packages/dara-core/dara/core/interactivity/plain_variable.py

```diff
@@ -89,6 +89,85 @@ def get(self, key: str):
         """
         return self.copy(update={'nested': [*self.nested, key]}, deep=True)
 
+    def sync(self):
+        """
+        Create an action to synchronise the value of this Variable with input value sent from the component.
+
+        ```python
+
+        from dara.core import Variable
+        from dara.components import Select
+
+        var = Variable('first')
+
+        Select(
+            value=var,
+            items=['first', 'second', 'third'],
+            onchange=var.sync(),
```

Comment:
var is already synced with the component as it is passed as value would it make more sense if we had a second var in this case? Same with UrlVariable

Reply:
Fair point, will add a second variable